tests/unit/test_state.py

```diff
@@ -18,6 +19,22 @@
 from tests.unit import factories
 
 
+@pytest.fixture(name="patch_juju_version_33")
+def patch_juju_version_33_fixture(monkeypatch: pytest.MonkeyPatch):
```

Comment:
```suggestion
@pytest.fixture(name="patch_juju_version_35")
def patch_juju_version_35_fixture(monkeypatch: pytest.MonkeyPatch):
```

Shouldn't it be shouldn't it be 35?

Reply:
It should be 33, updated thanks :)